### PR TITLE
Task: Create a project structure

### PR DESCRIPTION
**Pull request by Mincer2412**
Maven project structure based on maven-archetype-webapp

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>4.0.1</version>
+      <scope>provided</scope>
+    </dependency>
+
+  </dependencies>
+
+  <build>
+    <pluginManagement><!-- lock down plugins versions to avoid using Maven defaults (may be moved to parent pom) -->
```
Comment (imbirevkotlya) [inline pom.xml]: I think there is too much plugins here..... no?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.epam.firstproject</groupId>
+  <artifactId>firstproject</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>war</packaging>
+
+  <name>Ask more Maven Webapp</name>
```
Comment (imbirevkotlya) [inline pom.xml]: we don't need in these blocks: name and url, as far I remember, I have no experience in using them in production

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (imbirevkotlya) [inline pom.xml]: Why we use java 7 instead of maybe java 8?

**Comment by imbirevkotlya** [comment]
[Create a project structure](https://trello.com/c/DcR9iwMQ/1-create-a-project-structure)

File: pom.xml (hunk 2 above)
Comment (IlyasYOY) [reply pom.xml]: I think they can remove it or write something more suitable

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.epam.firstproject</groupId>
+  <artifactId>firstproject</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>war</packaging>
+
+  <name>Ask more Maven Webapp</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
```
Comment (IlyasYOY) [inline pom.xml]: Use github link

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
```
Comment (IlyasYOY) [inline pom.xml]: https://mvnrepository.com/artifact/junit/junit/4.12

use 4.12

I think this template is outdated

File: pom.xml (hunk 1 above)
Comment (IlyasYOY) [reply pom.xml]: I think it's fine

I would just removed the comments

File: pom.xml (hunk 3 above)
Comment (IlyasYOY) [reply pom.xml]: or maybe 11

File: target/Ask more/META-INF/MANIFEST.MF [hunk 6]
```diff
@@ -0,0 +1,5 @@
+Manifest-Version: 1.0
```
Comment (IlyasYOY) [inline target/Ask more/META-INF/MANIFEST.MF]: add `target` & `out` folders in .gitignore
And than it will be okay I think

File: pom.xml [hunk 7]
```diff
@@ -9,21 +9,21 @@
   <version>1.0-SNAPSHOT</version>
   <packaging>war</packaging>
 
-  <name>Ask more Maven Webapp</name>
-  <!-- FIXME change it to the project's website -->
-  <url>http://www.example.com</url>
+  <name>First-lab-project Maven Webapp</name>
```
Comment (IlyasYOY) [inline pom.xml]: remove `Maven Webapp` from name, not every OS supports spaces in folder names (as far as I know)

**Comment by IlyasYOY** [comment]
`target/Ask more` is still there, remove it and I approve this

**Pull request closed without merging**